### DISCUSSION TITLE
Could Django use Flet instead of Javascript?

### DISCUSSION THREAD

**Discussion by blaisep** [flet-dev/flet#1187] [Ideas]
(I've been a way for a while, big fan of this project)
Edit: expanded the title to refer to Django

I wonder if anyone has already incorporated Flet into Wagtail CMS or even Django?
When I was reading https://dev.to/lb/creating-a-schematic-editor-within-the-wagtail-cms-with-stimulusjs-n5j
it occurred to me that, since Wagtail is built on Django, maybe we could use Flet to render the HTML and deal with the UI instead of Javascript

**Comment by JorgeAlvarez21** [comment]
check out this repo https://github.com/Marysia-Software-Limited/flet-django

**Reply by blaisep** [reply]
Fantastic!
